docker-entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-if [ ! -f "/root/.config/supertuxkart/config-0.10/stkservers.db" ]; then
-    touch /root/.config/supertuxkart/config-0.10/stkservers.db
```

Comment:
If you remove this file, you won't be able any more to use a db file, isn't it? Have you tested it? Sorry I don't have time now to do it my self (I may in a few weeks)

Reply:
I will test things out this week :)